### ISSUE TITLE
Passing in expected values for test acceptance via JSON config

### ISSUE THREAD

**Issue by iniesC** [qca/boardfarm#37]
I would like to pass in expected values to the test being run. The tests can use these values to check the actual output with expected to report pass or fail. Does this support exists?

**Comment by wwahammy** [comment]
@iniesC I don't think functionality like that exists. I think it'd be a great addition. I think we could put it in the the JSON config file for example. Does that make sense do you think? Also, do you have a recommendation for the format?

**Comment by iniesC** [comment]
@ericschultz, The JSON format is fine. The goal is to map the testcase with a list of parameters that the test case could use. If no entry is provided in the JSON or no JSON file provided then we pass in defaults that can be used by the tests. Also, we don't have to update the module as we add new tests.

**Comment by wwahammy** [comment]
@iniesC I think that makes sense. I think we're on the same page here.

Could you expand upon your last sentence though? I'm not sure I understand what you mean by it.

Also, I'm going to update the issue title since this is a feature request.

**Comment by iniesC** [comment]
@ericschultz , I just want this new feature of accepting input values for tests to be robust and flexible in way that it wouldn't require updates when we add or remove tests from BFT.

**Comment by wwahammy** [comment]
@iniesC makes sense. I don't think I have time to work on this myself right now. If you feel comfortable doing a prototype or implementation, I'd be happy to help though.
